src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.Portable.cs

```diff
@@ -317,6 +317,8 @@ internal void WaitForRemoval()
 
     public static partial class ThreadPool
     {
+        internal const bool s_enableWorkerTracking = false;
```

Comment:
```suggestion
        internal static readonly bool s_enableWorkerTracking = false;
```
As a `const` C# gives unreachable code warnings

Reply:
Ah ok, thanks!

Reply:
We'll get a warning either way: if it's const, you might get compiler warnings about unreachable code, and if it's a static readonly, you'll get analyzer warnings about a static readonly being initialized to a constant value.  Another alternative is to make it a property:
```C#
internal static bool EnableWorkerTracking { get; } = GetEnableWorkerTracking();
```
on coreclr and:
```C#
internal static bool EnableWorkerTracking => false;
```
on mono.  But you could also just make it a const and suppress any compiler warnings.

Reply:
Thanks, I have suppressed the warning